week-2/F-logical-operators/exercise.js

```diff
@@ -13,13 +13,35 @@ var cssLevel = 4;
 // (hint: use the comparison operator from before)
 var htmlLevelAbove5;
 var cssLevelAbove5;
+if(htmlLevel > 5){
+  htmlLevelAbove5 = true;
+}
+else{
+htmlLevelAbove5 = false;
+}
```

Comment:
Can you write these without using any `if` statements? Have a look at the 2nd block of code under 'Logical operators' https://codeyourfuture.github.io/syllabus-london/js-core/week-05/lesson.html#logical-operators

Reply:
@LucyMac  sure I can, will  update it